lib/screens/pokemon_info/widgets/tab_base_stats.dart

```diff
@@ -112,6 +115,39 @@ class _PokemonBaseStatsState extends State<PokemonBaseStats> with SingleTickerPr
     ];
   }
 
+  String _removeTrailingZero(double n){
+    String s = n.toString().replaceAll(RegExp(r"([.]*0)(?!.*\d)"), "");
+    return s;
+  }
```

Comment:
this method should not belong to the `tab_base_stats.dart`, please move it to `lib/utils/string.dart`

Reply:
lib/utils/string.dart doesn't exist... Should I create it?